projects/admin/src/app/record/detail-view/circ-policy-detail-view/circ-policy-detail-view.component.html

```diff
@@ -80,6 +80,14 @@ <h1 class="mb-3">{{ record.metadata.name | translate }}</h1>
               {{ record.metadata.renewal_duration }}
               {{ record.metadata.renewal_duration | i18nPlural: {'=1': 'day', 'other': 'days'} | translate }}
             </dd>
+            <dt class="col-3 offset-4 label-title" translate>Automatic renewal</dt>
+            <dd id="cipo-auto-renewal" class="col-5 mb-0">
+              @if (record.metadata.automatic_renewal) {
+                <i class="fa fa-check text-success" aria-hidden="true"></i>
+              } @else {
+                <i class="fa fa-times text-danger" aria-hidden="true"></i>
+              }
```

Comment:
I think this syntax could work
`<i class="fa fa-check text-{{ record.metadata.automatic_renewal ? 'success' : 'danger' }}" aria-hidden="true">`</i>
To be tested

Reply:
It seems to create problems with the html tags `G5002: Unexpected closing tag "dl". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags`. Also, the icon is different not only the color so it is more readable to use an if.

Reply:
My bad:
`<i class="fa {{ record.metadata.automatic_renewal ? 'fa-check text-success' : 'fa-times text-danger' }}" aria-hidden="true"></i>`